src/api/order.js

```diff
@@ -88,7 +88,7 @@ const handleSmartResponse = (data : Object, headers : {| [$Values<typeof HEADERS
 };
 
 export function getOrder(orderID : string, { facilitatorAccessToken, buyerAccessToken, partnerAttributionID, forceRestAPI = false } : OrderAPIOptions) : ZalgoPromise<OrderResponse> {
-    if (forceRestAPI && !window[LSAT_UPGRADE_FAILED]) {
+    if (forceRestAPI && (window[LSAT_UPGRADE_FAILED] && !window[LSAT_UPGRADE_FAILED])) {
```

Comment:
Could we check for the exact boolean value like this?
```suggestion
    if (forceRestAPI && window[LSAT_UPGRADE_FAILED] === false) {
```